config/settings.py

```diff
@@ -158,3 +158,5 @@
 CSP_IMG_SRC = "'self' data:"
 
 REFERRER_POLICY = "same-origin"
+
+ADMIN_URL = os.getenv("ADMIN_URL")
```

Comment:
Que dirais-tu de mettre une valeur par défaut, histoire de se protéger si la variable d'environnement est manquante ?

```suggestion
ADMIN_URL = os.getenv("ADMIN_URL", "admin/")
```